packages/carbon-graphs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Fixed the goal line element in bar graph overlays the ability to click on the bar data.
```

Comment:
nit: `Fixed bug causing the goal line element to prevent clicking on the bar graph underneath.`

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/153/commits/026c5cc2a95a48b2a0329922394cb583c8aa44ab#diff-7b73f4b73c95577a16e8a58d2e1377b2204811d315c020884e80b9f22128c42aR6).